js/src/chrome/users/chrome.users.js

```diff
@@ -279,14 +279,21 @@ Wu.Chrome.Users = Wu.Chrome.extend({
 		});
 
 		// this._access[options.type]
-		var options = JSON.stringify({
+		var options = {
 			access : access
-		});
+		};
 
 		// create share link
-		Wu.post('/api/invite/link', options, function (a, b) {
-			this._shareLinkInput.value = b;
-		}.bind(this), this);
+		app.api.inviteLink(options, function (err, response) {
+			if (err) {
+				return app.feedback.setError({
+					title : 'Something went wrong in _createShareableInvite',
```

Comment:
error message too specific. Remember, these messages are for the user.

`Something went wrong`, is enough for user feedback. (Same for all other places where similar message.)